stablehlo/dialect/StablehloOps.cpp

```diff
@@ -477,6 +477,24 @@ LogicalResult CustomCallOp::verify() {
              << "operand part has type " << operandPart
              << " and output part has type " << outputPart;
   }
+
+  // Check backend_config attribute.
+  if (auto backendConfig = getBackendConfig()) {
+    if (getApiVersion() == CustomCallApiVersion::API_VERSION_TYPED_FFI) {
+      // Typed FFI custom calls require `backend_config` to be a DictionaryAttr.
+      if (isa<mlir::StringAttr>(*backendConfig))
```

Comment:
```
if (!isa<mlir::DictionaryAttr>(*backendConfig)) {
 // error
}
```

Directly cehck what is expected here?

Reply:
+1, and `!isa<StringAttr>` for the other branch